includes/class-ssp-frontend.php

```diff
@@ -442,12 +442,12 @@ public function get_album_art( $episode_id = false ) {
 		}
 
 		if ( $series_id ) {
-			$series_image = get_option( "ss_podcasting_data_image_{$series_id}", false );
+			$series_image = get_term_meta( $series_id, $this->token . '_series_image_settings', true );
```

Comment:
The series images we are using are those set in the Settings section of the plugin, under Feed Details, which is stored as options. However, I've discussed this with Craig and he agrees that we should also probably check against the term meta as a fallback for the Series image, before using the default Feed Image.

Essentially, check for the `ss_podcasting_data_image_{$series_id}` option for the image, if that returns nothing, use the `$this->token . '_series_image_settings' term meta for the image